intranet/apps/auth/urls.py

```diff
@@ -5,6 +5,8 @@
 urlpatterns = [
     re_path(r"^$", views.index_view, name="index"),
     re_path(r"^login$", views.LoginView.as_view(), name="login"),
+    re_path(r"^lockout", views.lockout_view, name="lockout"),
```

Comment:
I would recommend that `lockout` & `unlock`, go at `/ratelimit`

Reply:
So `re_path(r"^ratelimit/lockout", views.lockout_view, name="lockout"),`?

Reply:
Yes, (and same for `unlock`). You also need a trailing `$` to indicate the close of the path (since we are using `re_path` here).